CHANGELOG.md

```diff
@@ -10,6 +10,61 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 8.42.0
+
+### Important Changes
+
+- **feat(react): React Router v7 support (library) ([#14513](https://github.com/getsentry/sentry-javascript/pull/14513))**
+
+  This release adds support for [React Router v7 (library mode)](https://reactrouter.com/home#react-router-as-a-library).
+  Check out the docs on how to set up the integration: [Sentry React Router v7 Integration Docs](https://docs.sentry.io/platforms/javascript/guides/react/features/react-router/v7/)
+
+  Note that support for React Router v7 in framework mode is still being worked on.
+  Full support is planned to release early 2025.
```

Comment:
hmm, I'd tend to remove this sentence, we do not really have a time range defined yet, IMHO?

Reply:
Yeah makes sense. I thought we can rather confidently say early 2025 because we are low key blocked by v9 but maybe it's just limiting ourselves. I'll remove.